### ISSUE TITLE
Support for local-only delivery?

### ISSUE THREAD

**Issue by kevinoid** [corecode/dma#62]
Great project!  I am currently using Exim configured as a local-only MTA and would like to switch to dma.  I couldn't find a way to configure dma to disable remote delivery.  Is it currently possible?  If not, is it a feature which you might be willing to consider?  If you are don't plan to implement it, but would accept a PR, any guidelines for the configuration option name etc.?

Thanks for considering,
Kevin

**Comment by corecode** [comment]
how would handle mails with a remote destination?  bounce them?

**Comment by kevinoid** [comment]
Exim delivers an DSN to the local postmaster.  For my own uses, it would be fine if `sendmail` printed an error message and exited non-zero without queuing the message at all.  I'd defer to your judgement for whichever is better (or easier to implement).

**Comment by corecode** [comment]
Fair enough, why don't you submit a patch for a new config setting, say `LOCAL_ONLY`?

**Comment by tedkotz** [comment]
I think I will give this a try. Though, I think I will call the setting `LOCALONLY` to follow the convention of the other config items that do not have underscores. And I can easily subst it if a different name makes more sense.

Great project by the way. As the smallest mail server in Debian, I love it in embedded, and kiosk type setups.
I have wished it had local only delivery for a long time.

**Comment by tedkotz** [comment]
Actually I just saw this is duplicated by #121.

I'm going to push my change up to my fork, but hold off on making a PR.
